mava/components/jax/building/parameter_client.py

```diff
@@ -194,3 +194,171 @@ def on_building_trainer_parameter_client(self, builder: SystemBuilder) -> None:
     def name() -> str:
         """Component type name, e.g. 'dataset' or 'executor'."""
         return "trainer_parameter_client"
+
+
+####################
+# SEPARATE NETWORKS
+###################
+
+
+class ExecutorParameterClientSeparateNetworks(BaseParameterClient):
```

Comment:
What is the issue we face when trying to implement a general ParameterClient that works with both network setups?

Reply:
Discussed below :)